programmers-coding-test/minjae/Implement/구현-게임개발.js

```diff
@@ -0,0 +1,52 @@
+function solution(n) {
+    const mapField = [
+        [1, 1, 1, 1],
+        [1, 0, 0, 1],
+        [1, 1, 0, 1],
+        [1, 1, 1, 1],
+    ];
+    const dx = [-1, 0, 1, 0];
+    const dy = [0, 1, 0, -1];
+
+    let x = n[0];
+    let y = n[1];
+    let direction = n[2];
+
+    let nx, ny, turnTime;
+    nx = ny = turnTime = 0;
+
+    let count = 1; //처음 위치 카운트
+    mapField[x][y] = 2;
```

Comment:
map 의 본래 뜻이 지도이므로 map이라는 변수만 써도 좋을 것 같아요.
```suggestion
   map[x][y] = 2;
```